src/ui/components/UserDefinesList/index.tsx

```diff
@@ -39,6 +41,22 @@ interface UserDefinesListProps {
 
 const UserDefinesList: FunctionComponent<UserDefinesListProps> = (props) => {
   const { options, onChange, firmwareVersionData } = props;
+
+  const storage = new Application();
+  const [savedBindingPhraseHistory, setBindingPhraseHistory] = useState(['']);
+
+  useEffect(() => {
+    storage
+      .getBindingPhrase()
```

Comment:
This should be getBindingPhraseHistory instead of getBindingPhrase.

Reply:
ya, this was just an artifact of me trying to figure out id the issue was my getBindingPhraseHistory() function or otherwise. Definitely shouldn't have been committed.